eva-lib/src/main/java/uk/ac/ebi/eva/lib/repository/VariantEntityRepository.java

```diff
@@ -60,4 +63,11 @@ Long countByChromosomeAndStartAndReferenceAndAlternate(String chromosome, int st
 
     @Query(value = "{'chr': ?0, 'start': ?1, 'ref': ?2}", count = true)
```

Comment:
The queries don't work without the annotation. The reason is that it's querying for VariantEntity objects, but this class doesn't have document annotations for its variables since it's quite a basic subclass of the biodata variant object. The Mongo document is instead converted by the DBObjectToVariantEntityConverter class. 
The Variant class has different names for the alt and ref alleles than in the Mongo document ("alt" and "ref" in Mongo doc, "alternate" and "reference" in the Variant(Entity) class. So, for example, when a repository method is called "countByChromosomeAndStartAndRefAndAlt" Spring throws an error complaining:

"Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'variantEntityRepository': Invocation of init method failed; nested exception is org.springframework.data.mapping.PropertyReferenceException: No property ref found for type VariantEntity!"

and when the method is called "countByChromosomeAndStartAndReferenceAndAlternate" it just doesn't return anything. But it does work and return results with the Query annotation.